src/flicking.js

```diff
@@ -101,27 +102,72 @@ eg.module("flicking", ["jQuery", eg, window, document, eg.MovableCoord], functio
 				// jscs:enable validateLineBreaks, maximumLineLength
 			}
 
+			this._setOptions(options);
+			this._setConfig($children, _prefix);
+
+			!ns._hasClickBug() && (this._setPointerEvents = $.noop);
+
+			this._build();
+			this._bindEvents();
+
+			this._applyPanelsCss();
+			this._arrangePanels();
+
+			this.options.hwAccelerable && SUPPORT_WILLCHANGE && this._setHint();
+			this._adjustContainerCss("end");
+		},
+
+		/**
+		 * Set options values
+		 * @param {Object} options
+		 */
+		_setOptions: function(options) {
+			var padding = [0, 0];
+			var bounce = [10, 10];
+			var self = this;
+
+			var setDefault = function (name, defVal) {

```

Comment:
I think that it is ambiguous function name. `setDefault` is general name but It is not general working.